### PR TITLE
mdbook-pdf: 0.1.8 -> 0.1.10

### PR DESCRIPTION
**Pull request by HollowMan6**
## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [X] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [X] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by dotlambda — COMMENTED** [review]
Please remove rustfmt.

**Comment by HollowMan6** [comment]
> Please remove rustfmt.

I somehow remember it was necessary for the dependency `rust-headless-chrome`, but let's see how it goes now.

**Comment by dotlambda** [comment]
You also have to take care of `DO_NOT_FORMAT`.

**Comment by HollowMan6** [comment]
Ah okay, now I see the comments in the file, thanks!

File: pkgs/tools/text/mdbook-pdf/default.nix [hunk 1]
```diff
@@ -34,10 +32,8 @@ rustPlatform.buildRustPackage rec {
   # generate the Chrome Devtools Protocol
   DOCS_RS=true;
```
Comment (dotlambda) [inline pkgs/tools/text/mdbook-pdf/default.nix:33]:
```suggestion
  env.DOCS_RS = true;
```

File: pkgs/tools/text/mdbook-pdf/default.nix [hunk 2]
```diff
@@ -34,10 +32,8 @@ rustPlatform.buildRustPackage rec {
...
 
-  # # Stop formating with rustfmt, pending version update for
-  # # https://github.com/mdrokz/auto_generate_cdp/pull/8
-  # # to remove rustfmt dependency
-  # DO_NOT_FORMAT=true;
+  # Stop formating with rustfmt
+  DO_NOT_FORMAT=true;
```
Comment (dotlambda) [inline pkgs/tools/text/mdbook-pdf/default.nix:36]:
```suggestion
  # Stop formatting with rustfmt
  env.DO_NOT_FORMAT = true;
```

**Comment by HollowMan6** [comment]
> @HollowMan6 please merge it

I don't think I have the authority to merge.

image

**Comment by dotlambda** [comment]
Result of `nixpkgs-review pr 332655` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mdbook-pdf</li>
  </ul>
</details>

**Pull request merged**